### ISSUE TITLE
Manual and standalone modes need more description

### ISSUE THREAD

**Issue by jsha** [letsencrypt/letsencrypt#1181]
If you run the client without Apache installed, you get offered `manual`, and `standalone` modes. From context it's not obvious what these do:

```
How would you like to authenticate with the Let's Encrypt CA?
1  Manual Authenticator (manual)
2  Standalone Authenticator (standalone)
```

We should include a short explanation of each.

**Comment by bmw** [comment]
@Swartzcr actually fixed this today in #1079. Both `manual` and `standalone` have meaningful descriptions. `manual` is now hidden due to some UI/UX issues with it that need to be worked out, but if you run the client with the Apache/Nginx plugin installed, you'll see a much nicer description of `standalone`.

**Comment by pde** [comment]
Closing this ticket.  Feel free to open another one if you don't like the UI after Noah's change and #1182.
